layout/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.pkgproj

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <Import Project="$(MSBuildProjectName).props" />
+  
+  <PropertyGroup>
+    <IsLineupPackage Condition="'$(PackageTargetRuntime)' == ''">true</IsLineupPackage>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <SkipValidatePackage>true</SkipValidatePackage>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(PackageTargetRuntime)' == ''">
+    <ProjectReference Include="@(Project)" />
+    
+    <!-- Include refs -->
+    <File Include="$(CompileDirectory)*.*">
+      <TargetPath>ref/$(TargetFramework)</TargetPath>
+    </File>
+
+    <FilePackageDependency Include="Microsoft.NETCore.Platforms">
+      <Version>$(PlatformPackageVersion)</Version>
+    </FilePackageDependency>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(PackageTargetRuntime)' != ''">
+    <NativeFile Include="$(RuntimeDirectory)*.a" />
```

Comment:
RuntimeDirectory is set below I'm supprised this isn't always null. I thought Items outside of a target were always evaluated in order.